test/unit/services/s3.test.js

```diff
@@ -1,4 +1,10 @@
-const { listObjects } = require('../../../src/services/s3');
+const { 
```

Comment:
Looks like just this `s3.test.js` file is relevant to the PR. Can this PR be updated to only include this file?

The other code looks like it's for notes. You're free to keep them locally or in a separate branch!

Reply:
There are several ways to to go about it. `git reset --soft` is one method if you are brave enough for it 😉